### PR TITLE
BZ#1624573 - Fix issue where error message is not displayed for consoles

### PR DESCRIPTION
**Pull request by chalettu**
Fix for https://bugzilla.redhat.com/show_bug.cgi?id=1624573.   Error messages not being displayed properly for console launch errors. 
@miq-bot add_label gaprindashvili/yes
@miq-bot add_label bug

### CODE REVIEW

**Comment by chalettu** [comment]
Before SS
before

After SS
after

**Comment by chalettu** [comment]
Thanks @AllenBW .  Seems this one caused a test to fail.  Working on fixing it now.

File: client/app/services/consoles.service.js [hunk 1]
```diff
@@ -47,8 +47,15 @@ export function ConsolesFactory ($window, CollectionsApi, $timeout, $location, E
     }
   }
 
-  function consoleError (error) {
-    EventNotifications.error(__('There was an error opening the console. ' + error.message))
+  function consoleError (response) {
+    let message = __('undefined')
+    if (response.data) {
+      message = response.data.error.message
+    } else if (response.message) {
```
Comment (AllenBW) [inline client/app/services/consoles.service.js]: When would this case happen (the response.message)? I thought error messages were always of the `response.data.error.message` format?

Comment (chalettu) [reply client/app/services/consoles.service.js]: I dug through our test cases and it seems that in the use case that the response was a 200 and status was != success .  I am assuming the test cases were accurate as I didn't have a way to forcibly replicate this http response scenario.  This change I made supports that scenario and seems harmless enough in the case the scenario doesn't happen.

Comment (AllenBW) [reply client/app/services/consoles.service.js]: Hokay, yeah no harm, merging.

**Comment by JPrause** [comment]
@miq-bot add_label blocker

**Pull request merged**